chapters/functions.tex

```diff
@@ -2063,6 +2063,7 @@ \subsubsection{Records}\label{records}
 \end{itemize}
 
 Scalar records are passed by reference (i.e., a pointer to the record is being passed).
+Record inputs to C-functions are const-pointers, indicating that the records shall not be changed.
```

Comment:
We need to decide on which terminology to use; _by reference_ or _pointer_:
```suggestion
Scalar records are passed by reference (i.e., a pointer to the record is being passed).
Record inputs are passed by const reference, indicating that the records shall not be changed.
```

Alternatively:
```suggestion
Scalar records are passed as pointers (by reference).
Record inputs are const-pointers, indicating that the records shall not be changed.
```

Note that there is no need to say _to C-functions_ here, since the entire section is only about external C.